tools/issue-labeler/src/IssueLabelerService/AzureSdkIssueLabelerService.cs

```diff
@@ -69,49 +69,35 @@ public async Task<ActionResult> Run([HttpTrigger(AuthorizationLevel.Function, "P
                 return EmptyResult;
             }
 
+            TriageOutput result = new TriageOutput { Labels = labels.Values };
+
             if(bool.Parse(config.EnableAnswers))
             {
                 try{
 
-                    // Get the Qna model based on configuration
-                    var qnaService = _answerServices.GetAnswerService(config);
-
-                    var answer = await qnaService.AnswerQuery(issue, labels);
+                    // Get the Answer Service based on configuration
+                    var answerService = _answerServices.GetAnswerService(config);
 
-                    TriageOutput result = new TriageOutput
-                    {
-                        Labels = labels.Values,
-                        Answer = answer.Answer,
-                        AnswerType = answer.AnswerType
-                    };
+                    var answer = await answerService.AnswerQuery(issue, labels);
 
-                    return new JsonResult(result);
+                    result.Answer = answer.Answer;
+                    result.AnswerType = answer.AnswerType;
                 }
                 catch (Exception ex)
                 {
                     _logger.LogError($"Error commenting on issue #{issue.IssueNumber} in repository {issue.RepositoryName}: {ex.Message}{Environment.NewLine}\t{ex}{Environment.NewLine}");
 
-                    TriageOutput result = new TriageOutput
-                    {
-                        Labels = labels.Values,
-                        Answer = null,
-                        AnswerType = null
-                    };
-
-                    return new JsonResult(result);
+                    result.Answer = null;
+                    result.AnswerType = null;
                 }
             }
             else
```

Comment:
You can remove the `else`.    The `null` values are the default.   We included this explicitly in the `catch` block above just in case there was an error that occurred setting them, to ensure there's no inconsistent state (like answer being set but answer type being null).